src/main/entities/NewPassword.java

```diff
@@ -0,0 +1,146 @@
+package entities;
+
+public class NewPassword extends Password {
+
+    private int passlen;
+    private String choices;
+
+    public void setPasslen(int passlen) {
+        this.passlen = passlen;
+    }
+
+    public void setChoices(String choices) {
+        this.choices = choices;
+    }
+
+    public void generatePassword() {
+        if (choices.length() == 4) {
+            setPassword(lowUpSymNum(passlen));
```

Comment:
Why is lowUpSymNum not called fourChoices?

Reply:
No Specific reason, I developed his function first(before making the algorithm for 2/3 choices & hence named it a bit differently). Changed it.